### PR TITLE
Feature/tao 5148 test runner storage

### PR DESCRIPTION
**Pull request by krampstudio**
Implement the `testStore` in the QTI Test Runner and adapt the plugins.

Please note there's a remainder. 
The `testState` which is a `cachedStore` is not yet migrated to the testStore (only async).

**Requires** 
- [ ] https://github.com/oat-sa/tao-core/pull/1621
- [ ] https://github.com/oat-sa/extension-tao-test/pull/210

### CODE REVIEW

File: views/js/runner/provider/qti.js [hunk 1]
```diff
@@ -491,15 +501,8 @@ define([
...
                     self.dataUpdater.updatePluginsConfig(self.getPlugins());
 
-                    //check if we need to trigger a storeChange
-                    if(!_.isEmpty(storeId) && !_.isEmpty(results.lastStoreId) && results.lastStoreId !== storeId){
-
-                        /**
-                         * We have changed the local storage engine (could be a browser change)
-                         * @event runner/provider/qti#storechange
-                         */
-                        self.trigger('storechange');
```
Comment (no-chris) [inline views/js/runner/provider/qti.js]: Is it safe to remove this event? Or could this be considered a breaking change?

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -1762,6 +1762,6 @@ public function update($initialVersion) {
             $this->setVersion('18.9.5');
         }
 
-        $this->skip('18.9.5', '19.0.1');
+        $this->skip('18.9.5', '19.1.0');
```
Comment (jsconan) [inline scripts/update/Updater.php]: Version conflict with the manifest:
- manifest `20.0.0`
- update `19.1.0`

File: views/js/runner/provider/qti.js (hunk 1 above)
Comment (jsconan) [reply views/js/runner/provider/qti.js]: I think this is ok since the plugins that listened to this event have also been updated accordingly

**Comment by no-chris** [comment]
This breaks some plugin tests. Maybe the provider mock needs to be adapted?

**Pull request merged**